exonum/src/node/state.rs

```diff
@@ -703,10 +703,16 @@ impl State {
     /// Transaction is ignored if the following criteria are fulfilled:
     /// - transactions pool size is exceeded
     /// - transaction isn't contained in unknown transaction list of any propose
-    pub fn add_transaction(&mut self, tx_hash: Hash, msg: Box<Transaction>) -> Vec<(Hash, Round)> {
+    /// - transaction isn't a part of block
+    pub fn add_transaction(
+        &mut self,
+        tx_hash: Hash,
+        msg: Box<Transaction>,
+        // if tx is in some of propose or in a block,
```

Comment:
For me  it's more clear, with rustfmt `block` mode, when the comment is as close as possible with the place that it comments.